docs/DeveloperGuide.md

```diff
@@ -280,17 +377,169 @@ Use case ends.
 
 3a. The given index is invalid
 
-> 3a1. AddressBook shows an error message <br>
+>3a1. CMDo shows help message <br>
+> Use case resumes at step 2
+
+  
+### Use case: Find a task
+
+**MSS**
+
+1. User requests to find a task
+2. CMDo shows a list of tasks <br>
+>Use case ends.
+
+**Extensions**
+
+1a. The list is empty
+
+> Use case ends
+
+### Use case: List all tasks
+
+**MSS**
+
+1. User requests to list all tasks
+2. CMDo shows a list of tasks <br>
+>Use case ends.
+
+**Extensions**
+
+1a. The list is empty
```

Comment:
what happens then? i do suggest for all list is empy, instead of ending the use case, perhaps you guys can give some form of user feedbacks :)